### PR TITLE
Added missing required argument

### PR DESCRIPTION
**Pull request by stokic**

### CODE REVIEW

**Review by fulopattila122 — APPROVED** [review]
:+1:

File: src/Checkout/Drivers/SessionStore.php [hunk 1]
```diff
@@ -112,7 +112,7 @@ public function update(array $data)
 
         if (Arr::get($data, 'ship_to_billing_address')) {
             $shippingAddress = $data['billpayer']['address'];
-            $shippingAddress['name'] = $this->getShipToName();
+            $shippingAddress['name'] = $this->getShipToName($this->billpayer);
```
Comment (fulopattila122) [inline src/Checkout/Drivers/SessionStore.php:115]: Actually, this class has no `billpayer` attribute.

**Comment by stokic** [comment]
it's still required ;)

**Comment by fulopattila122** [comment]
it is :)

> it's still required ;)

**Pull request merged**